scss/_popover.scss

```diff
@@ -203,15 +202,18 @@
   &:empty {
     display: none;
   }
-
-  // Boosted mod: popover body special padding when following a popover header
-  + .popover-body {
-    padding-top: var(--#{$prefix}popover-header-padding-bottom);
-  }
-  // End mod
 }
 
 .popover-body {
-  // Boosted mod: no `padding`
+  padding: var(--#{$prefix}popover-body-padding-y) var(--#{$prefix}popover-body-padding-x);
   color: var(--#{$prefix}popover-body-color);
 }
+
+// Boosted mod: needed to reset styles for custom popover and make it customisable
+.popover:not(.custom-popover) .popover-header + .popover-body {
```

Comment:
It's not possible to do that. `.custom-popover` comes from the documentation as an example, it doesn't exist in the library. One can use whatever name as a class.

Reply:
@MewenLeHo Tried something in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2077